set-start.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+var save = process.env.npm_config_save
+
+if (save) {
+  var fileName = path.join('..', '..', 'package.json')
+  var file = require(fileName)
+
+  file.scripts = file.scripts || {}
+  file.scripts[ 'start' ] = 'hoodie'
+
+  fs.writeFile(fileName, JSON.stringify(file, null, 2),
+    function (err) {
+      if (err) return console.log(err)
+    })
+
+  console.log('Start your Hoodie app with "npm start"' + '\n')
+}
```

Comment:
can we please move this file into `bin/set-start.js`? And how about renaming the script to `bin/setup.js`?